portfolio/src/App.jsx

```diff
@@ -1,157 +1,145 @@
-  const handleChange = (e) => {
-    const name = e.target.name;
-    const value = e.target.value;
-    setData({...data, [name]: value})
-  }
+const headerOptions = [
+  {
+    name: "Projects",
+    href: "#projects",
+  },
+  {
+    name: "Experience",
+    href: "#experience",
+  },
+  {
+    name: "About Me",
+    href: "#about-me",
+  },
+  {
+    name: "Contact",
+    href: "#contact",
+  },
+];
 
-  const handleSubmit = (e) => {
-    e.preventDefault()
-    console.log(data)
-  }
+const projects = [
+  {
+    imgSrc: AmazonNavigatorImgSrc,
+    githubHref: "https://github.com/alla0810/Group9",
+    description:
+      "The AMAZON NAVIGATOR is designed for customers to view item reviews and pricing.",
+  },
+  {
+    imgSrc: PlantParenthoodImgSrc,
+    githubHref: "https://github.com/VictoriaSweet/scary-potato",
+    description:
+      "Plant Parenthood is the application to help the those who are new to the hobby or just maybe even a weekend plant-sitter or award winning growers.",
+  },
+  {
+    imgSrc: PasswordGeneratorImgSrc,
+    githubHref: "https://github.com/tfkjosh/Password-generator",
+    description:
+      "This application is designed for an employee to generate a random password based on criteria they've selected.",
+  },
+  {
+    imgSrc: TextEditorImgSrc,
+    githubHref: "https://github.com/tfkjosh/PWA-Challenge",
+    livePreviewHref: "https://pwa-challenge1-b508091ee337.herokuapp.com/",
+    description:
+      "This application features a number of data persistence techniques, insuring the application runs regardless of browser supported function.",
+  },
+];
+
+function App() {
   return (
     <>
       <header>
-        <div className="container m-auto px-4 py-6">
-          <div className="flex justify-between items-center">
-            <div>
-              <h1 className="font-bold text-xl">Joshua's Portfolio</h1>
-            </div>
-          </div>
+        <div className="flex container m-auto px-4 py-6 justify-between items-center">
+          <h1 className="font-bold text-xl">Joshua's Portfolio</h1>
           <ul className="flex gap-4">
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Projects">
-                {" "}
-                Projects
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Experience">
-                {" "}
-                Experience
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#About-me">
-                {" "}
-                About Me
-              </a>
-            </li>
-            <li>
-              <a className="text-gray-400 hover:text-white cursor-pointer" a href="#Contact">
-                {" "}
-                Contact
-              </a>
-            </li>
+            {headerOptions.map(({ href, name }) => (
```

Comment:
It's always a good idea whenever you have repeating elements in the UI to use a `map` function to iterate over a list or array structure and then populate the same UI element with the data via that `map` function. Pretty much just makes it easier to add new elements to the UI by just having to add elements to the list instead of doing a bunch of copying and pasting of JSX code. It also guarantees that all of the repeated elements have the same exact styling. This has caught me a bunch in the past where I've left something out of one element in a list of repeated elements and then couldn't seem to figure out why only one of the elements had different styling than the rest